gnomad/utils/vcf.py

```diff
@@ -1167,24 +1168,32 @@ def make_hist_bin_edges_expr(
 
     :param ht: Table containing histogram variant annotations.
     :param hists: List of variant histogram annotations. Default is HISTS.
+    :param row_name: Name of row annotation containing histogram data. In exomes or
+        genomes release HT, `histograms` is a row, but in the joint releast HT, it's
+        under the row of `exomes`, `genomes`, or `joint`.
     :param prefix: Prefix text for age histogram bin edges.  Default is empty string.
     :param label_delimiter: String used as delimiter between prefix and histogram annotation.
     :param include_age_hists: Include age histogram annotations.
-    :return: Dictionary keyed by histogram annotation name, with corresponding reformatted bin edges for values.
+    :return: Dictionary keyed by histogram annotation name, with corresponding
+        reformatted bin edges for values.
     """
     # Add underscore to prefix if it isn't empty
     if prefix != "":
         prefix += label_delimiter
 
     edges_dict = {}
+    if row_name and hasattr(ht[row_name], "histograms"):
```

Comment:
Maybe this answers my question above, does ht[None] fail? Though my gut is if None was passed this would short circuit and `hasattr(ht[None],...)` wouldnt be evaluated

Reply:
I tried to use ht[None], it won't work. You meant something like this? 
```commandline
    row_name: Optional[str] = None,
...
                        ht.head(1)[row_name].histograms.age_hists[f"age_hist_{call_type}"]

```

Reply:
Yes, can you explain how it didnt work? I'd expect the `if row_name` to evaluate to False and `hasattr(..` wouldnt even be evaluated so your second line would never be an issue with None. But perhaps hail is getting angry about it when its creating the DAG.

Reply:
This error: 
`TypeError: Table.__getitem__: invalid index argument(s)
  Usage 1: field selection: ht['field']
  Usage 2: Left distinct join: ht[ht2.key] or ht[ht2.field1, ht2.field2]`